contents/newsletter/talk-to-users.md

```diff
@@ -0,0 +1,106 @@
+---
+title: An engineer's guide to talking to users
+date: 2024-04-18
+author:
+  - ian-vanagas
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/newsletter/beyond-10x-engineer/super-hog.png
+featuredImageType: full
+tags:
+  - Engineering
+  - Product engineers
+crosspost:
+  - Product engineers
+  - Blog
+---
+
+Talk to users? Isn't that a job for founders, product managers, user researchers, and salespeople? Why do would an engineer ever want or need to do it?
+
+In his book [*Modern Software Engineering*](https://www.davefarley.net/?p=352), Dave Farley writes that mastering software engineering requires becoming an expert in learning. When learning is prioritized, talking to users becomes a critical part of an engineer's toolkit.
+
+To explain further why this is the case, and to help you get started, we wrote this guide based on what we've learned talking to users as engineers ourselves at PostHog.
```

Comment:
Maybe more like...

So that begs the question: how should engineers be talking to users? Fret not, we've assembled an essential guide that covers everything you need to know.